src/imio/urban/core/actionspanel/views.py

```
@@ -48,6 +50,23 @@ def __call__(self,
+                # abort because element was partially removed
+                transaction.abort()
+                breaches = ILinkIntegrityInfo(self.request).getIntegrityBreaches()
+                linked_titles = []
+                for _unused, sources in breaches.items():
+                    for source in sources:
+                        title = source.Title()
+                        if isinstance(title, str):
+                            title = title.decode('utf-8')
+                        linked_titles.append(title)
+                if linked_titles:
+                    warning = u'Attention, cet \xe9l\xe9ment est utilis\xe9 dans les dossiers suivants : {0}'.format(
```

Ruff (0.15.12)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)